cvat-ui/src/components/login-page/cookie-policy-drawer.tsx

```diff
@@ -0,0 +1,50 @@
+
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React, {useState, useEffect} from 'react';
+import Drawer from 'antd/lib/drawer';
+import Paragraph from 'antd/lib/typography/Paragraph';
+import Button from 'antd/lib/button/button';
+
+import {isPublic} from 'utils/enviroment';
+
+
+function CookieDrawer(): JSX.Element {
+    const [drawerVisible, setDrawerVisible] = useState(false);
+
+    useEffect(() => {
+        const cookiePolicyAccepted = localStorage.getItem('cookiePolicyAccepted');
+        if (cookiePolicyAccepted === null && isPublic()) {
+            setDrawerVisible(true);
+        }
+    }, [isPublic])
```

Comment:
Useless construction
1) isPublic is a const and it never changes
2) Only component props can be specified here. External variables are legal, but if external variable changes, it doesn't mean that component will be rerendered. It works only for props.